IDEAS/Buildings/Data/Materials/GypsumBoard.mo

```diff
@@ -0,0 +1,17 @@
+within IDEAS.Buildings.Data.Materials;
+record GypsumBoard = IDEAS.Buildings.Data.Interfaces.Material (
+    k=0.16,
```

Comment:
![image](https://user-images.githubusercontent.com/5970879/179937468-dacfb608-957f-4b1e-8342-ce3b223f9291.png)

close enough! and a lot closer than the old value of 0.6..